server/services/bookingService.test.ts

```diff
@@ -136,7 +137,13 @@ describe('BookingService', () => {
             html: `<strong class="govuk-tag">Provisional</strong>`,
           },
           {
-            html: `<a href="#">View<span class="govuk-visually-hidden"> booking for person with CRN ${booking1.person.crn}</span></a>`,
+            html: `<a href="${taPaths.bookings.show({
```

Comment:
Is it worth merging these paths now we don't need to worry about seperating AP and TA?

Reply:
Think I want to just delete AP paths, now that there's only a couple of functions using it, and I can delete those now that the AP controllers are gone. Hopeful I can sneak that in this sprint!